docs/DeveloperGuide.md

```diff
@@ -765,11 +765,95 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
+### Creating an event
+1. Creating an event
+   1. Test case: `ecreate n/Food packing r/10 chef r/10 packer sd/23/10/2023 1900 ed/23/10/2023 2200 l/hougang dsc/Packing food for the needy m/100 packets b/100.00 vs/50`<br>
+      Expected: A new event is created and added to the event list panel. Details of the event are shown in the status message.
+
+   1. Test case: `ecreate`<br>
+      Expected: No change to the event list panel. Invalid command format error is shown in the status message.<br>
+      The outcome is the same when any of the compulsory parameters are missing from the input command.
+
+   1. Test case: `ecreate n/Food packing r/10 chef r/10 packer sd/23/10/2023 1900 l/hougang dsc/`<br>
+      Expected: No change to the event list panel. Error message for the invalid field is shown in the status message<br>
+      The outcome is the same when all compulsory parameters are present and any argument is empty or invalid.
+
+   1. Test case: `ecreate n/Food packing r/10 chef r/10 packer sd/23/10/2023 1900 l/hougang dsc/Packing food for the needy`, followed by
+                 `ecreate n/Food packing r/10 chef r/10 packer sd/23/10/2023 1900 l/hougang dsc/Packing food for the needy`<br>
+      Expected: No change to the event list panel. Error message for duplicate events is shown in the status message<br>
+      The outcome is the same when all compulsory parameters are present, all arguments are valid, and an event with the same name already exists in the event list.
+   
+### Adding a volunteer to an event
+1. Adding a volunteer which is being shown to an event which is being shown
+
+   1. Prerequisites: The event list must contain at least one event and the volunteer list must contain at least one volunteer.
```

Comment:
i think this can be event list panel and volunteer list panel instead

Reply:
true